README.md

```diff
@@ -91,11 +102,23 @@ First, add your credentials:
   - In the name field, put `AWS_ACCESS_KEY_ID` and inside the value, the access key id you had from earlier
   - Similarly, the second secret is named `AWS_SECRET_ACCESS_KEY` with the corresponding value.
 
-    More info about Gihub secrets [here](https://docs.github.com/en/actions/security-guides/encrypted-secrets)
+    More info about Gihub secrets [in the github documentation](https://docs.github.com/en/actions/security-guides/encrypted-secrets)
+
+### No Custom domain setup:
+
+**IMPORTANT** If you chose to move forward without a custom domain name, you need to do the following before you start:
+
+- Navigate to the GitHub Action for global tools at .github/workflows/deploy-global-tools.yaml and **remove** the line starting with `TFVARS_AUTO_JSON`
```

Comment:
The other folders/files have \` around them, can you add them here as well, same with the `deploy-dev...`below